<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrainSensor</class>
 <widget class="QWidget" name="StrainSensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>应变传感器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(38, 183, 248)</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>831</width>
     <height>361</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>传感器名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传感器型号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传感器状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>生产厂家</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传感器规格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传感器编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>生产日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>传感器波长</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>671</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>传感器名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSenName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>传感器型号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditModel"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditStatue">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>传感器生产厂家</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditManufa"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>传感器规格</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSpftion"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>传感器编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNum"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>生产日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>传感器波长</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditWavelenth"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSave">
      <property name="text">
       <string>保存信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDel">
      <property name="text">
       <string>删除信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="text">
       <string>添加信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="returnpushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上一层</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
